### PR TITLE
add matrix shield to readme

### PR DESCRIPTION
**Pull request by oraNod**
Adds the matrix shield

### CODE REVIEW

**Review by cybette — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-(code for Matrix shield, etc, goes here)
+[![Discuss at #website:ansible.com on Matrix](https://img.shields.io/matrix/community:ansible.com.svg?server_fqdn=ansible-accounts.ems.host&label=Discuss%20at%20%23website:ansible.com&logo=matrix)](https://matrix.to/#/#website:ansible.com)
```
Comment (GregSutcliffe) [inline README.md:1]: That URL is wrong, you have `community:ansible.com` in there.

Comment (GregSutcliffe) [reply README.md:1]: The link works but it's saying 183 users instead of the 60ish we have

Comment (cybette) [reply README.md:1]: Ah, looks like if you update it to `website:ansible.com` it will show the correct number (33)

Comment (GregSutcliffe) [reply README.md:1]: Huh, I could have sworn it was higher, but yes, that's the change I mean :)

**Review by GregSutcliffe — APPROVED** [review]
LGTM

**Review by cybette — APPROVED** [review]

**Pull request merged**